libmachine/mcnutils/b2d.go

```diff
@@ -20,12 +19,8 @@ var (
 	GithubApiToken string
 )
 
-const (
-	timeout = time.Second * 5
-)
-
 func defaultTimeout(network, addr string) (net.Conn, error) {
-	return net.DialTimeout(network, addr, timeout)
+	return net.Dial(network, addr)
```

Comment:
Introduced in `d1ee81453ca393acce39e16ece3474fd3fb234cc` by @ehazlett, I think to increase resilience in case of network failure?  The fallback is the image cache AFAIK.